content/v2.0/security/tokens/_index.md

```diff
@@ -15,6 +15,10 @@ weight: 101
 InfluxDB ensures secure interaction between users and data through the use of **authentication tokens**.
 A token belongs to an organization and identifies InfluxDB permissions within the organization.
 
+Use authentication tokens for the following tasks:
+- Authorize API requests. See [InfluxDB v2 API](/v2.0/reference/api/#view-influxdb-v2-api-documentation).
+- Write data using the Influx CLI. See [influx CLI](/v2.0/write-data/#influx-cli)
```

Comment:
Providing this context is tricky because 1. writing to InfluxDB is done through the API so this list is redundant and 2. almost all API interactions require authentication. There are only a handful that do not.

Reply:
@noramullen1 @sanderson -this one is tricky. 

Content suggestions to start below...and work thru together:

On the [Manage authentication tokens page](https://v2.docs.influxdata.com/v2.0/security/tokens/), line 18:
To successfully write data to InfluxDB, you must include an auth....in your write request.
Select one of the following ways of writing data to see how to add your authentication token:
- [CLI](https://v2.docs.influxdata.com/v2.0/get-started/#using-the-influx-cli-after-setting-up-influxdb-through-the-ui) 
- [Telegraf](https://v2.docs.influxdata.com/v2.0/write-data/use-telegraf/manual-config/#example-influxdb-v2-configuration)
- [API](https://v2.docs.influxdata.com/v2.0/write-data/#influxdb-api) (perhaps here, we can add a list of API requests that don’t require authentication? @sanderson, do we already have these doc'd?)

On the Write Data to InfluxDB page—in this section: https://v2.docs.influxdata.com/v2.0/write-data/#influx-cli, we could add a note that if token hasn’t already been set up, see https://v2.docs.influxdata.com/v2.0/get-started/#using-the-influx-cli-after-setting-up-influxdb-through-the-ui. Alternatively, we may consider moving this information to the “Write Data to InfluxDB” page-given folks may follow the UI to “get started” and then not think to refer to the “getting started” docs), expecting to find w/in context of "writing data”. One to noodle.